base/weakkeydict.jl

```diff
@@ -48,10 +48,10 @@ WeakKeyDict() = WeakKeyDict{Any,Any}()
 WeakKeyDict(kv::Tuple{}) = WeakKeyDict()
 copy(d::WeakKeyDict) = WeakKeyDict(d)
 
-WeakKeyDict{K,V}(ps::Pair{K,V}...)            = WeakKeyDict{K,V}(ps)
-WeakKeyDict{K  }(ps::Pair{K}...,)             = WeakKeyDict{K,Any}(ps)
-WeakKeyDict{V  }(ps::(Pair{K,V} where K)...,) = WeakKeyDict{Any,V}(ps)
-WeakKeyDict(     ps::Pair...)                 = WeakKeyDict{Any,Any}(ps)
+WeakKeyDict(ps::Pair{K,V}...)            where {K,V} = WeakKeyDict{K,V}(ps)
+WeakKeyDict(ps::Pair{K}...)              where {K  } = WeakKeyDict{K,Any}(ps)
+WeakKeyDict(ps::(Pair{K,V}  where K)...) where {V  } = WeakKeyDict{Any,V}(ps)
```

Comment:
The two spaces in `Pair{K,V}  where` seems odd? Perhaps shift that space (for alignment I guess?) to before the second `where`?